scripts/createSchema.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+import { schema } from '@kiwicom/margarita-graphql';
+import { printSchema } from 'graphql';
+import fs from 'fs';
+import path from 'path';
+
+fs.writeFileSync(
+  path.join(__dirname, '..', 'schema.graphql'),
+  printSchema(schema),
+);
```

Comment:
🤔 https://github.com/kiwicom/margarita/blob/06455a179940dc4688029d05a131701eb65bf504/scripts/relay-compiler.js#L20

The reason why I added it together with Relay Compiler is that it always fetches (reads) new schema before Relay works with it so you don't even have to think about such a step. This script should not be necessary.

Reply:
Yes, I was unaware of this when making the PR. I will remove it.